interface/d3.network.js

```diff
@@ -0,0 +1,679 @@
+/* Reusable d3 force-directed layout for gene networks
+ * Follows the design pattern suggested by Mike Bostock:
+ * http://bost.ocks.org/mike/chart/
+ */
```

Comment:
Ideally the `d3.network` library should be modified in my fork and reviewed by Aaron before it is merged to his master repo. But when we started to add more and more customization to our gene-gene network, it doesn't seem to be practical to ask Aaron to review every change. So I decided to use my own fork in `index.html` a few months ago.

But the issue that Deb had yesterday made me realize that it was not a good long-term solution, because:
(1) Whatever change I made to `d3.network.js` should be reviewed by somebody else.
(2) It makes more sense to bundle `d3.network.js` into our release instead of loading it from github. That way any previous version of adage-server doesn't have to be dependent on the one on github.

That is why I am including `d3.network.js` in this PR. Eventually I would like to change its coding style to the one we use and file another PR in the future. Then our development of this library will be independent from Aaron's repo.